python/arcticdb/version_store/library.py

```diff
@@ -940,6 +940,56 @@ def read_metadata(self, symbol: str, as_of: Optional[AsOf] = None) -> VersionedI
         """
         return self._nvs.read_metadata(symbol, as_of)
 
+    def read_metadata_batch(
+        self, symbols: List[Union[str, ReadRequest]], query_builder: Optional[QueryBuilder] = None
+    ) -> List[VersionedItem]:
+        """
+        Reads the metadata of multiple symbols.
+
+        Parameters
+        ----------
+        symbols : List[Union[str, ReadRequest]
+            List of symbols to read.
+            Params columns, date_range and query_builder from ReadRequest are not used
+        
+        query_builder: This parameter is not used
+
+        Returns
+        -------
+        List[VersionedItem]
+            A list of the read results, whose i-th element corresponds to the i-th element of the ``symbols`` parameter.
+
+        See Also
+        --------
+        read
+        """
+        symbol_strings = []
+        as_ofs = []
+        def handle_read_request(s_):
+            symbol_strings.append(s_.symbol)
+            if s_.date_range is not None:
+                warnings.warn("Read batch metadata does not make use of of the data_range field", SyntaxWarning, stacklevel=2)
```

Comment:
Why this `warnings` approach rather than our normal logging? How does `warnings` interact with logger configuration?